plugins/SitesManager/Controller.php

```diff
@@ -250,7 +251,7 @@ public function getTrackingMethodsForSite()
         ];
 
         $recommendedMethod = null;
-        $matomoIndex = null;
+        $matomoIndex = $matomoTagManagerIndex = null;
```

Comment:
I often miss when multiple variables are assigned a value on a single line of code, no harm in keeping it on separate rows.

```suggestion
        $matomoIndex = null;
        $matomoTagManagerIndex = null;
```